src/Components/Home.js

```diff
@@ -2,21 +2,37 @@ import './Home.css';
 import { useState } from 'react';
 
 const Home = () => {
-    const [value, setValue] = useState('');
+    const initialValue = [];
+    const [value, setValue] = useState([]);
+    const [initial, setInitial] = useState(false);
     var val = '';
     const onChange = (e) => {
         val = e.target.value;
-        console.log(val);
+        if (initial == true) {
+            e.target.value = "";
+            setInitial(false);
+        }
     }
-    const onClickHandler = () => {
-        setValue(val);
+    const onClickHandler = (e) => {
+        if (val != "") {
+            setValue([...value, val]);
+            //console.log(e.target.value, "val");
+            setInitial(true);
+        }
     }
-    return(
+    const onComplete = (e) => {
+        //console.log(e.target.innerText);
+        const arr = value.filter((item) => item !== e.target.innerText);
+        setValue(arr);
+    }
+    return (
         <div className="todo">
             <h1>Todo-List</h1>
             <input className="inputVal" type="text" onChange={onChange} placeholder="Enter the Task...." /> &nbsp;
-            <button className="btn" onClick={onClickHandler}>Add Task</button>
-            <p className="todo-task">{value}</p>
+            <button className="btn" onClick={onClickHandler}>Add Task</button><br/><br/>
+            {value.map((user) => (
+                <div className="todo-task" onClick={onComplete} >{user}</div>
```

Comment:
There must be a warning in the console. Fix that:-
`Warning: Each child in a list should have a unique "key" prop.`